### ISSUE TITLE
it's not working to extract single package consisting of tiny packages.

### ISSUE THREAD

**Issue by frogred8** [arei/npmbox#59]
it is a pretty good job for single package feature.
i tested multiple packages into single package.
common packages is working well but tiny multiple packages is not working.

```
bash> npmbox npm async express node 
bash> npmunbox npm.npmbox
```

it's ok 

```
bash> npmbox is type-is cookie
bash> npmunbox is.npmbox
```

it's not working well (extracting only one package "is")

```
bash> ls -al node_modules 
bash> total 0
bash> drwxr-xr-x   3  263744041  102 Aug 15 15:37 .
bash> drwxr-xr-x  12  263744041  408 Aug 15 15:37 ..
bash> drwxr-xr-x  10  263744041  340 Aug 15 15:37 is
```

i think "getTargets()" function is called too early before it was completed to extract packages.
(it may be tar.gz library issue )
so my temporary solution is to call the function late.

```
setTimeout(function() {
   getTargets();
}, 500);
```

i know it is not the right way perfectly.
if you have a better way, apply that you think.

**Comment by danfuzz** [comment]
I believe this is fixed by my PR #66, which has been merged but not yet published to the npm registry.

**Comment by danfuzz** [comment]
Closing this as presumed-fixed by v4.0.2 (just published). Please comment / reopen if you find that the new version is still causing you trouble.
